src/card/pages/CardCompletePage.tsx

```diff
@@ -1,47 +1,53 @@
 import { ChangeEvent, useState } from 'react';
-import { AppLayout, CardDisplay, useCard, useFunnel } from '@/components';
-import { Button, HStack, TextField, Typography, VStack } from '@/shared/components';
-import { removeAllSpaces } from '@/shared/utils';
+import { CardDisplay, useCard } from '@/card';
+import { AppDisplay, Button, HStack, TextField, Typography, VStack, removeAllSpaces, useFunnel } from '@/shared';
 
-const CARD_DESCRIPTION_MAX_LENGTH = 20;
+const CARD_DESCRIPTION_MAX_LENGTH = 10;
 
 export const CardCompletePage = () => {
   const { goToNext } = useFunnel();
-  const { card, addCardToOwner } = useCard();
+  const { card, addCardToOwner, editCardToOwner, isCardExist } = useCard();
   const {
-    label: cardBrandLabel,
+    label: cardBrandName,
     color: cardBrandColor,
     cardNumber,
     expirationDate: cardExpirationDate,
     ownerName: cardOwnerName,
+    description,
   } = card;
 
-  const [cardDescription, setCardDescription] = useState('');
+  const [cardDescriptionInputValue, setCardDescriptionInputValue] = useState<string>(description ?? '');
+
   const handleCardDescriptionChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setCardDescription(e.target.value.trim());
+    setCardDescriptionInputValue(e.target.value.trim());
   };
-  const isValidateCardDescription = removeAllSpaces(cardDescription).length > 0;
 
-  const handleCardCreateSubmit = () => {
-    addCardToOwner({
-      ...card,
-      description: cardDescription,
-    });
+  const completeCardRegistry = () => {
+    const description = removeAllSpaces(cardDescriptionInputValue) ? cardDescriptionInputValue : cardBrandName;
+
+    if (isCardExist(card)) {
+      editCardToOwner({ ...card, description });
+    } else {
+      addCardToOwner({
+        ...card,
+        description,
+      });
+    }
```

Comment:
```suggestion
    if (isCardExist(card)) {
      editCardToOwner({ ...card, description });
    } else {
      addCardToOwner({ ...card, description });
    }
```
일관성 있는 포맷으로 보여주면 좋을 것 같아요!

Reply:
> 일관성 있는 포맷으로 보여주면 좋을 것 같아요!

prettier로 제안주신 포맷이 개선되면 좋겠다 생각되네요!
룰을 한번 찾아보고 가능하다면 적용해보겠습니다. 꼼꼼하게 컨벤션을 챙겨주셔서 감사합니다. 👍